### PR TITLE
Add two files test1 and test2

### PR DESCRIPTION
**Pull request by phammongkha**
### Tasks

- [x] Task1

- [x] Task2

### Test report

- Test report 1

### CODE REVIEW

File: test1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Develop branch  - Commit 1
```
Comment (PMKha) [inline test1.txt]: @phammongkha 
Please add comment here!

File: test2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+develop branch - test2 commit
```
Comment (PMKha) [inline test2.txt]: Please add more information!

File: test1.txt (hunk 1 above)
Comment (phammongkha) [reply test1.txt]: Thank you for review!

File: test2.txt (hunk 2 above)
Comment (phammongkha) [reply test2.txt]: Thanks so much for your information

**Comment by phammongkha** [comment]
It's been fixed!

**Comment by phammongkha** [comment]
Thank you for reviewing carefully. I'll merge this PR.

**Pull request merged**